src/empty-state.css.ts

```diff
@@ -0,0 +1,93 @@
+import {style, createVar, styleVariants} from '@vanilla-extract/css';
+import * as mq from './media-queries.css';
+import {sprinkles} from './sprinkles.css';
+import * as skin from './skins/skin-contract.css';
+
+const backgroundColor = createVar();
+
+export const vars = {backgroundColor};
+
+export const container = style([
+    sprinkles({
+        display: 'flex',
+        backgroundColor: 'transparent',
+    }),
+    {
+        flexDirection: 'column-reverse',
+
+        '@media': {
+            [mq.tabletOrSmaller]: {
+                border: 'none',
+                maxWidth: 368,
+            },
+            [mq.desktopOrBigger]: {
+                borderRadius: 16,
+                flexDirection: 'row',
+                overflow: 'hidden',
+                backgroundColor,
+            },
+        },
+    },
+]);
+
+export const contentVariants = styleVariants({
+    default: {
+        marginTop: 0,
+        '@media': {
+            [mq.desktopOrBigger]: {
+                margin: 64,
+            },
+        },
+    },
+    largeImage: {
```

Comment:
I found differences in image behavior in mobile

production: [example 1](https://ibit.ly/lbCm) | [example 2 (1:1)](https://ibit.ly/5ibK)
this branch: [example 1](https://ibit.ly/JbUR) | [example 2 (1:1)](https://ibit.ly/Ezj_)

maybe missing `width 100%`?

I think the correct behavior should not show a deformed image, just should fit the photo to the longest side like the image component → https://ibit.ly/FHan

In desktop sizes seem to behave well

Reply:
@yceballost changes image behaviour test again in mobile and desktop (to be sure all still works as expected:) )